### ISSUE TITLE
asus x00rd

### ISSUE THREAD

**Issue by murdika** [phhusson/treble_experimentations#688]
camera didnt work,scrooling screen broken,plz fix

**Comment by phhusson** [comment]
Aosp v119?

**Comment by murdika** [comment]
Aosp v10 on my ASUS_X00RD

**Comment by phhusson** [comment]
What is magisk camera? This doesn't sound related to this ROM?
What does "scroll screen broken" mean? perhaps you want to make a photo?

**Comment by kukuruyukpagi** [comment]
Same here, Lenovo S5 K520/seoul. Every build version booted but always had visual glitch when scroll. Camera not working even tried other cam apps.

**Comment by murdika** [comment]
plz fix camera doesnt work on my asus live l1..

Pada tanggal Rab, 18 Sep 2019 2.50 AM, Pierre-Hugues HUSSON <
notifications@github.com> menulis:

**Comment by N1ght420** [comment]
> Same here, Lenovo S5 K520/seoul. Every build version booted but always had visual glitch when scroll. Camera not working even tried other cam apps. Vendor 8.0

No visual glitch on vendor 9.0, but you can try to updating your gpu driver by flashing this .zip file
https://drive.google.com/file/d/1aNaBLv9fmIcurRK1LdxQnO83_JKfij7O/view?usp=sharing

My phone is Lenovo S5 K520 too, and visual glitch are fixed by flashing that .zip file

**Comment by kukuruyukpagi** [comment]
> > Same here, Lenovo S5 K520/seoul. Every build version booted but always had visual glitch when scroll. Camera not working even tried other cam apps. Vendor 8.0
> 
> No visual glitch on vendor 9.0, but you can try to updating your gpu driver by flashing this .zip file
> https://drive.google.com/file/d/1aNaBLv9fmIcurRK1LdxQnO83_JKfij7O/view?usp=sharing
> 
> My phone is Lenovo S5 K520 too, and visual glitch are fixed by flashing that .zip file

Yeah, i just forgot to update this issue. visual glitch solved by update adreno driver (vendor blob only). and the camera also has been fixed on v204, at least on my Lenovo S5 K520.
